spanner/batch.go

```diff
@@ -150,13 +150,14 @@ func (t *BatchReadOnlyTransaction) PartitionReadUsingIndexWithOptions(ctx contex
 	}
 	// Prepare ReadRequest.
 	req := &sppb.ReadRequest{
-		Session:        sid,
-		Transaction:    ts,
-		Table:          table,
-		Index:          index,
-		Columns:        columns,
-		KeySet:         kset,
-		RequestOptions: createRequestOptions(readOptions.Priority, readOptions.RequestTag, ""),
+		Session:          sid,
+		Transaction:      ts,
+		Table:            table,
+		Index:            index,
+		Columns:          columns,
+		KeySet:           kset,
+		RequestOptions:   createRequestOptions(readOptions.Priority, readOptions.RequestTag, ""),
+		DataBoostEnabled: readOptions.DataBoostEnabled,
```

Comment:
Add a one line comment: 
`If this is for a partitioned read and this field is set to true, the request will be
executed via Spanner independent compute resources. `

Reply:
I have added it under ReadOptions
https://github.com/googleapis/google-cloud-go/pull/8152/files#diff-98c680ad0e9a6842d77932c6b37ee80a322d5b5e462a26a4f0154844ea4e8513R169